cropper/app/lib/Crops.scala

```diff
@@ -21,8 +21,8 @@ object Crops {
   import scala.concurrent.ExecutionContext.Implicits.global
   import Files._
 
-  def outputFilename(source: SourceImage, bounds: Bounds, outputWidth: Int, isMaster: Boolean = false): String = {
-    s"${source.id}/${Crop.getCropId(bounds)}/${if(isMaster) "master/" else ""}$outputWidth.jpg"
+  def outputFilename(source: SourceImage, bounds: Bounds, outputWidth: Int, exportType: String, isMaster: Boolean = false): String = {
+    s"${source.id}/${Crop.getCropId(bounds)}/$exportType/${if(isMaster) "master/" else ""}$outputWidth.jpg"
```

Comment:
Note the change in output paths. This means paths won't be backwards compatible. Not entirely sure if it's worth distinguishing images in their path, rather than having them flat (there is still a `type` metadata property recorded on each S3 object), so open for discussion.